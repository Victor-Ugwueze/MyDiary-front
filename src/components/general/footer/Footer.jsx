// react libraries
import React from 'react';

const Footer = () => (
  <footer>
      <p>Copywright &copy; My Dairy 2018</p>
  </footer>
);

export default Footer;
