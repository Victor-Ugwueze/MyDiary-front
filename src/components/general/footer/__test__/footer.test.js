// React Libraries
import React from 'react';

// third-party libraries
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';


// components
import Footer from '../Footer';


configure({
  adapter: new Adapter(),
});

describe('Renders the footer', () => {
  it('should render the footer of landing page', () => {
    const component = shallow(
      <Footer />
    );
    expect(component).toMatchSnapshot();
  });
});
