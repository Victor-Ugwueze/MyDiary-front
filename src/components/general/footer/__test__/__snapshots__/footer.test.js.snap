// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the footer should render the footer of landing page 1`] = `
<footer>
  <p>
    Copywright Â© My Dairy 2018
  </p>
</footer>
`;
